src/Sentry/SentryFailedRequestHandler.cs

```diff
@@ -0,0 +1,100 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Sentry.Protocol;
+
+namespace Sentry
+{
+    internal class SentryFailedRequestHandler : ISentryFailedRequestHandler
+    {
+        private readonly IHub _hub;
+        private readonly SentryOptions? _options;
+
+        public string MechanismType { get => "SentryFailedRequestHandler"; }
+
+        /// <summary>
+        /// Initializes an instance of <see cref="SentryFailedRequestHandler"/>.
+        /// </summary>
+        internal SentryFailedRequestHandler(IHub hub, SentryOptions? options)
+        {
+            _hub = hub;
+            _options = options;
+        }
+
+        /// <summary>
+        /// Automaticlaly capture HTTP Client errors when captureFailedRequests
+        /// </summary>
+        /// <param name="request">The HttpRequestMessage sent</param>
+        /// <param name="response">The HttpResponse received</param>
```

Comment:
Some typos here also.  But again, you can just remove it since the class is internal.